src/ui/icons/PoisonMicroblockIcon.tsx

```diff
@@ -0,0 +1,26 @@
+'use client';
```

Comment:
We could also use cube icon from react-icons for this one.
<img width="335" alt="image" src="https://github.com/hirosystems/explorer/assets/97111756/1c4e91f4-788b-4ffe-babe-60cc784e786b">